perl/holdings_proxy.cgi

```diff
@@ -33,7 +33,7 @@ use Cwd 'abs_path';
 use File::Basename qw(dirname);
 
 my $cmd_path = dirname(abs_path($0));
-my $config_ref = do("$cmd_path/holdings_proxy.config");
+my $config_ref = do("$cmd_path/holdings_proxy_esa.config"); # HUOM.
```

Comment:
The path to the config file must be `"$cmd_path/holdings_proxy.config")` in version control. You can change the path on the server when testing.